feat.[006-swagger-and-utils]

gopythor opened the pull request:
### 변경사항
<!-- 이 PR에서 어떤점들이 변경되었는지 기술해주세요. 가급적이면 as-is, to-be를 활용해서 작성해주세요.  -->
**AS-IS**
스웨커 기본 설정 및 Boolean Y-N 변환
Aes암호 변환 유틸

**TO-BE**


### 테스트
<!-- 본 변경사항이 테스트가 되었는지 기술해주세요 --> 
- [ ] 테스트 코드
- [x] API 테스트

gopythor committed:
feat.[001-member-entity]
멤버 엔티티 초기 설정

gopythor committed:
feat.[002-member-sign-up]
멤버 토큰 가입(일반)

gopythor committed:
feat.[003-member-sign-in]
멤버 토큰 로그인(일반)

gopythor committed:
feat.[004-jwt-token]
jwt토큰 관련
id(Long)와 email이 포함

gopythor committed:
feat.[005-custom-exception]
커스텀 익셉션 구현
ErrorCode에 에러코드 추가 가능

gopythor committed:
feat.[006-swagger-and-utils]
스웨커 기본 설정 및 Boolean Y-N 변환
Aes암호 변환 유틸

71mC40H0 commented:
리뷰 드렸습니다. 다른 커밋과 겹치는 부분은 그 쪽에서 드렸고요. Code Convention 정해지면 그거에 맞게 정리 해주시면 될 것 같습니다.

gopythor committed:
refactoring.[007-member]
active 컬럼 삭제
imageUrl로 수정
nickname 추가

gopythor committed:
refactoring.[008-member-sign-up]
param 부분 최적화
ㄴ 이미 param으로 매개변수명이 주소창으로 넘어감

gopythor committed:
refactoring.[008-member-sign-up]
param 부분 최적화
ㄴ 이미 param으로 매개변수명이 주소창으로 넘어감

위의 문제로 수정했으나, 매핑의 모호함에러로 인해 컴파일이 안됨
재 수정
